fpga/zedboard/leds/runVivadoLeds.tcl

```diff
@@ -0,0 +1,54 @@
+# TCL script automating synthesis, implementation and programming of a Zedboard
```

Comment:
Hmm yes, but I decided to keep the LED project entirely outside of the CMake build system because it is just to simple and does not really have anything to do with FlexPRET. It is there mostly just so the user can easily verify that their FPGA is working as expected.

I instead wrote in the README how to use the project. But it is a little inconsistent. Do you think it would be better to integrate it?

Reply:
That makes sense, lets keep it!